test/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDaoTest.java

```diff
@@ -54,21 +54,23 @@
 @ContextConfiguration("classpath:test-context.xml")
 @RunWith(SpringJUnit4ClassRunner.class)
 public class DynamoSurveyDaoTest {
-
-    private static Logger logger = LoggerFactory.getLogger(DynamoSurveyDaoTest.class);
+    private static final Logger logger = LoggerFactory.getLogger(DynamoSurveyDaoTest.class);
```

Comment:
by convention everywhere else we've leaned on calling this LOG I think (or maybe LOGGER). It's fine with me if we standardize on this as we came across it.

Reply:
Renamed to LOG.